__tests__/app.test.js

```diff
@@ -114,12 +114,79 @@ describe("GET /api/articles", () => {
       expect(articles).toBeSortedBy('created_at', { descending: true });
     });
   });
-  test("400: Responds with Bad Request: Invalid Query when passed an invalid sort_by or order_by query ", () => {
+  // test("400: Responds with Bad Request: Invalid Query when passed an invalid sort_by or order_by query ", () => {
+  //   return request(app)
+  //   .get("/api/articles?sort_by=name")
+  //   .expect(400)
+  //   .then(({ body })=>{
+  //     expect(body.message).toBe('Bad Request: Invalid Query')
+  //   });
+  // });
+});
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of comments for the given article_id, sorted by most recent first", () => {
     return request(app)
-    .get("/api/articles?sort_by=name")
-    .expect(400)
-    .then(({ body })=>{
-      expect(body.message).toBe('Bad Request: Invalid Query')
-    });
+      .get("/api/articles/9/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments }  = body;
+        expect(comments).toHaveLength(2)
+        comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("body");
+          expect(comment).toHaveProperty("article_id");
+        });
+        expect(comments[0]).toMatchObject({
+          body: "Oh, I've got compassion running out of my nose, pal! I'm the Sultan of Sentiment!",
+          votes: 16,
+          author: "butter_bridge",
+          article_id: 9,
+        });
+        expect(comments[1]).toMatchObject({
+          body: "The owls are not what they seem.",
+          votes: 20,
+          author: "icellusedkars",
+          article_id: 9,
```

Comment:
I like your thinking here where you've tested both comments, however the .forEach, kind of does the job. On line 141, you could assert that they both have the ID of 9, which would do a similar job to what you're doing in each comment check.